runtimes/8.0/Dockerfile

```diff
@@ -40,6 +40,22 @@ RUN apt-get update \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+ARG XDEBUG
+ARG XDEBUG_PORT
+RUN if [ "${XDEBUG}" = 'true' ]; then \
+    apt-get update \
+    && apt-get install -y php-xdebug \
+    && echo "[XDebug]" > /etc/php/8.0/cli/conf.d/php-ext-xdebug.ini \
+    && echo "zend_extension="$(find /usr/lib/php/ -name xdebug.so | sort -z | head -n 1)" >> /etc/php/8.0/cli/conf.d/php-ext-xdebug.ini" \
```

Comment:
there is a typo with double-quotes, also enclosing value in quotes for a safe path
```
&& echo "zend_extension = \"$(find /usr/lib/php/ -name xdebug.so | sort -z | head -n 1)\"" >> /etc/php/8.0/cli/conf.d/php-ext-xdebug.ini \
```